src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -113,6 +114,13 @@ public String getServicePhraseName(Interface service) {
     return Name.upperCamel(service.getSimpleName()).toPhrase();
   }
 
+  /** Artifact ID of the service */
+  public String getArtifactId(Interface service) {
+    List<String> tokens =
+        Splitter.on('.').splitToList(service.getModel().getServiceConfig().getName());
+    return tokens.size() > 0 ? "google-cloud-" + tokens.get(0) : null;
```

Comment:
You are hardcoding the knowledge of google-cloud. This doesn't work for non-google-cloud services. We need to get this from config somehow.